collector/test/SystemInspectorServiceTest.cpp

```diff
@@ -1,10 +1,9 @@
-#include "SysdigService.h"
-#include "gmock/gmock.h"
```

Comment:
Interesting, it wasn't used here?

Reply:
It wasn't, the tests use full fledged Falco objects instead of mocks, they just doesn't instruct the inspector to start collecting data.